src/components/Welcome/WelcomeContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import { useNavigate, useParams } from 'react-router-dom';
```

Comment:
`useParams` causes a compilation error because it isn't used.  Unless there is intention to use this hook, it should be removed to prevent this compilation error.